Kitodo/src/main/java/org/goobi/production/flow/statistics/hibernate/StatQuestProduction.java

```diff
@@ -79,22 +78,17 @@
         // retrieving the statistical data
         Integer exactStepDone = null;
         String stepname = null;
-        List<DataTable> allTables = new ArrayList<>();
 
         // gathering some information from the filter passed by dataSource
         // exactStepDone is very important ...
 
-        //TODO; find way to replace it
-        /*try {
-            exactStepDone = originalFilter.stepDone();
-        } catch (UnsupportedOperationException e1) {
-            logger.error(e1);
-        }
-        try {
-            stepname = originalFilter.stepDoneName();
-        } catch (UnsupportedOperationException e1) {
-            logger.error(e1);
-        }*/
+        // TODO; find way to replace it
```

Comment:
Why your are formatting out commented code? If this is not needed more then remove it or do not touch it. It is unreadable after your formatting.

Reply:
The code formatter reformats the whole file. Commented-out code should be removed, as it is not used any more.

Reply:
Then remove the out commented code.